internal/json/step.go

```diff
@@ -0,0 +1,52 @@
+package json
+
+import "strconv"
+
+// Step represents a step in the path to a field that should be stripped from
+// the input data.
+type Step struct {
+	Step   string
+	Filter []Filter
+}
+
+// Filter represents a filter that should be validated before a Field is
+// stripped.
+type Filter struct {
+	Path  []string
```

Comment:
Could you please add a test case where a Filter struct's Path has more than one element? I'm finding it hard to understand how/when that would be used

Reply:
[Done!](https://github.com/hashicorp/terraform-equivalence-testing/pull/32/commits/778b0b17fbef961bc69c8087b575f788272e7f56). Let me know if that doesn't cover what you wanted!